app/command/put_publish_intent.rb

```diff
@@ -0,0 +1,17 @@
+class Command::PutPublishIntent < Command::BaseCommand
+  attr_reader :event
+
+  def call
+    url_arbiter.reserve_path(
+      base_path,
+      publishing_app: content_item[:publishing_app]
+    )
+
+    live_content_store.put_publish_intent(
+      base_path: base_path,
+      publish_intent: content_item
+    )
+  rescue GOVUK::Client::Errors::HTTPError => e
```

Comment:
We shouldn't assume that that's the case. The plan was (still might be) to migrate all the api clients to separate gems with a shared base class.  The url-arbiter client was a first pass at this, and as such, these error classes were intended to be returned by all clients in time.